integration_tests/tests/in_cluster.rs

```diff
@@ -47,103 +48,169 @@ impl InClusterJanusPair {
+                    role: "Leader".to_string(),
+                    name: "leader".to_string(),
+                    api_url: Self::in_cluster_aggregator_api_url(&leader_namespace).to_string(),
+                    dap_url: Self::in_cluster_aggregator_dap_url(&leader_namespace).to_string(),
+                    bearer_token: leader_aggregator_api_auth_token,
+                };
+
+                let paired_leader_aggregator = divviup_api_pair_aggregator(
+                    &divviup_api_port_forward,
+                    &account,
+                    &leader_aggregator_pair_req,
+                )
+                .await;
+                paired_leader_aggregator.id
+            }
         };
-        let leader_task = leader_task
-            .with_aggregator_endpoints(leader_endpoints)
-            .build();
-        let leader = InClusterJanus::new(
-            &cluster,
-            &leader_namespace,
-            &leader_api_namespace,
-            &leader_task,
+
+        // We create a divviup-api account for each test, so we always need to pair the helper.
+        let helper_aggregator_pair_req = NewAggregatorRequest {
+            role: "Helper".to_string(),
+            name: "helper".to_string(),
+            api_url: Self::in_cluster_aggregator_api_url(&helper_namespace).to_string(),
+            dap_url: Self::in_cluster_aggregator_dap_url(&helper_namespace).to_string(),
+            bearer_token: helper_aggregator_api_auth_token,
+        };
+        let paired_helper_aggregator = divviup_api_pair_aggregator(
+            &divviup_api_port_forward,
+            &account,
+            &helper_aggregator_pair_req,
         )
         .await;
 
-        let helper_endpoints = {
-            let mut endpoints = helper_task.aggregator_endpoints().to_vec();
-            endpoints[0] = Self::in_cluster_aggregator_url(&leader_namespace);
-            endpoints
+        let provision_task_request = NewTaskRequest {
+            name: "Integration test task".to_string(),
+            leader_aggregator_id,
+            helper_aggregator_id: paired_helper_aggregator.id,
+            vdaf: task.vdaf().try_into().unwrap(),
+            min_batch_size: task.min_batch_size(),
+            max_batch_size: match task.query_type() {
+                QueryType::TimeInterval => None,
+                QueryType::FixedSize { max_batch_size } => Some(*max_batch_size),
+            },
+            expiration: "3000-01-01T00:00:00Z".to_owned(),
+            time_precision_seconds: task.time_precision().as_seconds(),
+            hpke_config: STANDARD.encode(task.collector_hpke_config().get_encoded()),
         };
-        let helper_task = helper_task
-            .with_aggregator_endpoints(helper_endpoints)
+
+        // Provision the task into both aggregators via divviup-api
+        let provisioned_task =
+            divviup_api_create_task(&divviup_api_port_forward, &account, &provision_task_request)
+                .await;
```

Comment:
Here, we'd have to make a new request to `GET /api/tasks/:task_id/collector_auth_tokens` as the collector auth token won't be in the provision task response.